integrations/google-analytics/test/index.test.js

```diff
@@ -1341,6 +1341,45 @@ describe('Google Analytics', function() {
           analytics.assert(window.ga.args[1][0] === 'set');
         });
 
+        it('should map user-defined position properties for products in products array', function() {
+          analytics.track('Product List Viewed', {
+            category: 'cat 1',
+            list_id: '1234',
+            products: [
+              {
```

Comment:
I missed this before, sorry - can we include multiple products in the products array? This will ensure we're properly testing that all products are passed to Google with the correct position.

Reply:
Done.